kuma/users/jinja2/users/user_edit.html

```diff
@@ -135,16 +135,19 @@ <h2>{{ _('Subscriptions') }} <a class="subscription-link" href="{{ url('recurrin
 
         </ul>
       </fieldset>
-      <p>
+      <div class="save-and-delete">
           <button type="submit" class="positive">
               {% include 'includes/icons/general/check-mark.svg' %}
               {{ _('Publish') }}
           </button>
-      </p>
-      {% if is_me %}
-      <a class="field-management" href="{{ url('users.user_delete', username=request.user.username) }}">{{ _('Delete your account.') }}</a>
-      {% endif %}
+          {% if is_me %}
+            <a href="{{ url('users.user_delete', username=request.user.username) }}">{{ _('Delete your account.') }}</a>
+          {% endif %}
+      </div>
     </form>
+    {% if is_me and request.user.is_staff and settings.STRIPE_PUBLIC_KEY %}
```

Comment:
```suggestion
    {% if is_me and waffle.flag('subscription') and settings.STRIPE_PUBLIC_KEY and settings.STRIPE_PLAN_ID %}
```

I think that's it. The `waffle.flag()` stuff knows about the user automagically. 
Next, you (and every dev) need to go into the Django admin and manually create a Waffle Flag that is named exactly this. 
Because that's so tedious, error prone, and annoying I would strongly recommend you ship this PR with a data migration that adds one. There are some good examples of this. You create the file like this: `docker-compose exec web ./manage.py makemigrations users --empty` and then you need to add something like this: https://github.com/mdn/kuma/blob/master/kuma/core/migrations/0003_auto_20191212_0638.py

That way, every dev gets the Waffle Flag spelled correctly, including stage and prod. 

that example `0003_auto_20191212_0638.py` uses `Flag.objects.filter(name='bc-signals').delete()` but should probably do...
```python
Flag.objects.create(name='subscription', super="conservative", options="that", you="and", kadir='feel', is='safe')
```

Reply:
Awesome, thanks for the help. I've added a migration for it.